sfepy/terms/terms_multilinear.py

```diff
@@ -1516,8 +1519,16 @@ def get_function(self, mat, virtual, mode=None, term_mode=None,
             )
 
         else:
+            dim = virtual.dim
+            if mat.shape[-2:] == (1, 1):
+                mul = '00'
+            elif mat.shape[-2:] in [(dim, 1), (dim, dim)]:
+                mul = 'jk'
+            else:
+                raise ValueError(f'wrong matrial shape! ({mat.shape[-2:]})')
+
             fun = self.make_function(
-                '00,i.j', mat, virtual, diff_var=diff_var,
+                mul + ',i.j', mat, virtual, diff_var=diff_var,
```

Comment:
It might be more readable to either put the entire `make_function()` call into the `if` branches (like other terms do), or at least use something like `expr = '00,i.j'`, ...